### ISSUE TITLE
Convert Print output to  Something Else 

### ISSUE THREAD

**Issue by metaboulie** [marimo-team/marimo#1262]
### Description

<img width="819" alt="Screenshot 2024-04-29 at 20 47 02" src="https://github.com/marimo-team/marimo/assets/114637443/da364c9e-f0d2-4735-b761-8c82d88e172d">

### Suggested solution

I'm using a `summary()` function from [pygam](https://github.com/dswah/pyGAM), but it automatically print the result out.
It turns out that they use `print()` in the [source code](https://github.com/dswah/pyGAM/blob/a6c14e48c34310567fdea1698735081b834b742d/pygam/pygam.py#L1790), which means I have only 2 choices:
1. In marimo, somehow control the output for a cell, and convert the output to something else can be better visualized like `mo.ui.table()`.
2. Fork the repo and replace `print` with something else.

**Comment by mscolnick** [comment]
You can redirect the print to the output and optionally modify it beforehand. You should check out: `mo.capture_stdout`

https://docs.marimo.io/api/outputs.html#marimo.capture_stdout

Given it's a string, it'll be hard to parse it back into a table.
